### PR TITLE
Add ruby example

### PR DESCRIPTION
**Pull request by mbj**
Adds the ruby example. I mailed you the full 100% covering solution in a gist to avoid spoilers.

### CODE REVIEW

File: ruby/Gemfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'mutant',             '~> 0.7.5', git: 'https://github.com/mbj/mutant.git'
```
Comment (mbj) [inline ruby/Gemfile:3]: @filipvanlaenen There is a bug in the last mutant release that can cause the reporter to crash, its fixed in master. I'll make a release before monday, expect a followup PR to bump the version here. (Maybe give me commit access to the repo).

**Comment by computas-fvl** [comment]
How do I give you full write access, so that I don't have to merge the pull requests?

**Comment by mbj** [comment]
@computas-fvl You should be able to add me using the following URL https://github.com/computas-fvl/oop2015/settings/collaboration Github will ask you for your password and than present you a form where you can add collaborators, enter my handle `mbj` and I should have push access.

**Comment by computas-fvl** [comment]
I know, but you already are a collaborator to the project, so I was a bit surprised I got a pull request.

Anyway, I got to go to the airport now. See you tomorrow morning.

**Comment by mbj** [comment]
Wow, I did not knew I'm a collaborator already. Sorry for sending PRs than. Cya!

**Pull request merged**